mappings/net/minecraft/world/storage/RegionFile.mapping

```diff
@@ -11,9 +11,11 @@ CLASS net/minecraft/class_2861 net/minecraft/world/storage/RegionFile
 	METHOD <init> (Ljava/io/File;Ljava/io/File;Z)V
 		ARG 1 file
 		ARG 2 directory
+		ARG 3 synchronous
 	METHOD <init> (Ljava/nio/file/Path;Ljava/nio/file/Path;Lnet/minecraft/class_4486;Z)V
 		ARG 1 file
 		ARG 2 directory
+		ARG 4 synchronous
```

Comment:
This actually triggers [dsync](https://docs.oracle.com/javase/8/docs/api/java/nio/file/StandardOpenOption.html#DSYNC) standard open options flag than sync (only content is guaranteed, file metadata is not), dunno if this had better be `synchronous` or `dsync`

Reply:
updated